docs/pages/mintlist/[mintlistAddress].tsx

```diff
@@ -0,0 +1,240 @@
+import { useNetworkContext } from "../../components/NetworkContext";
+import { useRouter } from "next/router";
+import { Solana } from "@glow-app/solana-client";
+import { Network } from "@glow-app/glow-client";
+import useSWR, { SWRResponse } from "swr";
+import { NftokenFetcher } from "../../utils/NftokenFetcher";
+import React from "react";
+import { SolanaAddress } from "../../components/SolanaAddress";
+import { ExternalLink } from "../../components/ExternalLink";
+import { ChevronLeftIcon, ExternalLinkIcon } from "@heroicons/react/outline";
+import { DateTime } from "luxon";
+import { ResponsiveBreakpoint } from "../../utils/style-constants";
+import { useGlowContext } from "@glow-app/glow-react";
+import { LuxButton } from "../../components/LuxButton";
+
+type ATTRIBUTE_KEY = keyof NftokenTypes.MintlistInfo;
+type ATTRIBUTE_TYPE = "address" | "link" | "unix_timestamp" | "amount";
+const KEYS: { key: ATTRIBUTE_KEY; type?: ATTRIBUTE_TYPE }[] = [
+  { key: "address", type: "address" },
+  { key: "authority", type: "address" },
+  { key: "treasury_sol", type: "address" },
+  { key: "go_live_date", type: "unix_timestamp" },
+  { key: "created_at", type: "unix_timestamp" },
+  { key: "metadata_url", type: "link" },
+  { key: "collection", type: "address" },
+  { key: "price", type: "amount" },
+  { key: "minting_order" },
+  // We will handle the ones below manually
+  // { key: "num_nfts_total" },
+  // { key: "num_nfts_redeemed" },
+];
+
+export default function MintlistPage() {
+  const { query } = useRouter();
+  const mintlistAddress = query.mintlistAddress as Solana.Address;
+
+  const { user } = useGlowContext();
+
+  const { network } = useNetworkContext();
+
+  const { data: mintlist } = useMintlist({ address: mintlistAddress, network });
+
+  return (
+    <>
+      <PageLayout>
+        {mintlist && (
+          <>
+            {user && (
+              <div className="navigation">
+                <LuxButton
+                  label="Back to Mintlists"
+                  icon={<ChevronLeftIcon />}
+                  href="/mintlists"
+                  iconPlacement="left"
+                  rounded
+                  variant="link"
+                  color="brand"
+                />
+              </div>
+            )}
+            <h1>{mintlist.name}</h1>
+            <div className="columns">
+              <div className="collection">
+                {mintlist.collectionInfo && (
+                  <>
+                    <h2>Collection</h2>
+                    <figure>
+                      <img
+                        alt={mintlist.collectionInfo.name}
+                        src={mintlist.collectionInfo.image}
+                      />
+                      <figcaption>{mintlist.collectionInfo.name}</figcaption>
+                    </figure>
+                  </>
+                )}
+              </div>
+
+              <div>
+                <h2>On-Chain Data</h2>
+                <div className="table">
```

Comment:
Let's also pull `ValueList` into a component. We also want more vertical spacing between rows. (And there is no divider.)